cirq/ops/common_gates_test.py

```diff
@@ -38,18 +59,7 @@ def test_cz_str():
     assert str(cirq.CZ**-0.25) == 'CZ**-0.25'
 
 
-def test_cz_repr():
```

Comment:
I wouldn't get rid of thi  particular repr test, because it is asserting more than just consistency. It's asserting that you get a *simple* repr for these cases, instead of the fully detailed one.